spearhead/assets/sass/_posts-and-pages.scss

```diff
@@ -27,11 +27,22 @@
 	}
 }
 
-.site-main > * {
-	margin: 0 auto;
-	padding: var(--global--spacing-vertical) 0;
-	&:first-child {
-		padding-top: 0;
+.site-main {
+	& > * {
+		margin: 0 auto;
+		padding: var(--global--spacing-vertical) 0;
+		&:first-child {
+			padding-top: 0;
+		}
+	}
+	article > * {
+		margin-top: calc( 0.48 * var(--global--spacing-vertical) );
+		margin-bottom: calc( 0.48 * var(--global--spacing-vertical) );
+ 
+	 	@include media(mobile) {
+			margin-top: calc( 0.48 * var(--global--spacing-vertical) );
+			margin-bottom: calc( 0.48 * var(--global--spacing-vertical) );
+	 	}
```

Comment:
It's a shame we can't combine the two selectors so we don't have to declare the rules twice. I feel like this might be an area of Seedlet we can improve.

Reply:
Maybe we can add a comment to explain why this code looks redundant.